src/js/helpers/apiRequest.js

```diff
@@ -20,12 +20,20 @@ export const apiRequest = (axiosParams = {}) => {
         return defaultHeaders;
     };
 
-    const params = (options = {}) => {
-        const parameters = { ...defaultParams };
-        Object.assign(parameters, options);
-        parameters.headers = headers(options.headers);
-        return parameters;
-    };
+    const params = (options = {}) => Object
```

Comment:
What made you want to use reduce here? Couldn't we just `if (params.data && params.award_id) params.data.award_id = decodeURIComponent(options.data.award_id);`

Reply:
You're right it just makes it more confusing to use reduce.